web/renderer/components/Navbar/index.tsx

```diff
@@ -4,9 +4,25 @@ import { ExternalLink, Navbar } from "@dolthub/react-components";
 import { dockerHubRepo, workbenchGithubRepo } from "@lib/constants";
 import { FaDocker } from "@react-icons/all-files/fa/FaDocker";
 import { FaGithub } from "@react-icons/all-files/fa/FaGithub";
+import css from "./index.module.css";
+
+const forElectron = process.env.NEXT_PUBLIC_FOR_ELECTRON === "true";
+
+const handleDoubleClick = () => {
+  window.ipc.macTitlebarClicked();
+};
 
 export default function Nav() {
-  return (
+  return forElectron ? (
+    <div className={css.titlebar} onDoubleClick={handleDoubleClick}>
+      <Navbar
+        logo={<Logo />}
+        leftLinks={<LeftLinks />}
+        rightLinks={<RightLinks />}
+        bgColor="bg-storm-600"
+      />
+    </div>
+  ) : (
```

Comment:
Since the `Navbar` component is repeated below I'd factor this out into a const
```suggestion
  const navbar =  (<Navbar
        logo={<Logo />}
        leftLinks={<LeftLinks />}
        rightLinks={<RightLinks />}
        bgColor="bg-storm-600"
      />)
  return forElectron ? (
    <div className={css.titlebar} onDoubleClick={handleDoubleClick}>
       {navbar}
    </div>
  ) : (
```